### PR TITLE
Restoring changes lost in dev merge from TANK-232-User-Card-Search-Bar-Parity

### PR DESCRIPTION
**Pull request by wmitch34**
Made this branch to restore my changes and ensure a clean merge. There is very little on it.

### CODE REVIEW

File: frontend/src/pages/dashboardtabs/Clients.tsx [hunk 1]
```diff
@@ -24,15 +24,18 @@ export default function Clients() {
...
     () => optionsList?.find((client) => client.id === selectedClientId) ?? null,
     [optionsList, selectedClientId]
   );
+  console.log('selectedClient on load: ', selectedClient);
 
   const handleUserSelected = (
     _event: React.SyntheticEvent,
     client: UserOption | null
   ) => {
+    console.log('client.id on handleUserSelected: ', client?.id);
```
Comment (thomas-harmeyer) [inline frontend/src/pages/dashboardtabs/Clients.tsx]: log

File: frontend/src/pages/dashboardtabs/Clients.tsx [hunk 2]
```diff
@@ -24,15 +24,18 @@ export default function Clients() {
   const [tankModalOpen, setTankModalOpen] = useState(false);
   const [clientModalOpen, setClientModalOpen] = useState(false);
   const [selectedClientId, setSelectedClientId] = useState<number | null>(null);
+  console.log('selectedClientId on load: ', selectedClientId);
   const selectedClient = useMemo(
     () => optionsList?.find((client) => client.id === selectedClientId) ?? null,
     [optionsList, selectedClientId]
   );
+  console.log('selectedClient on load: ', selectedClient);
```
Comment (thomas-harmeyer) [inline frontend/src/pages/dashboardtabs/Clients.tsx]: log

File: frontend/src/pages/dashboardtabs/Clients.tsx [hunk 3]
```diff
@@ -24,15 +24,18 @@ export default function Clients() {
   const [tankModalOpen, setTankModalOpen] = useState(false);
   const [clientModalOpen, setClientModalOpen] = useState(false);
   const [selectedClientId, setSelectedClientId] = useState<number | null>(null);
+  console.log('selectedClientId on load: ', selectedClientId);
```
Comment (thomas-harmeyer) [inline frontend/src/pages/dashboardtabs/Clients.tsx]: log

File: frontend/src/pages/dashboardtabs/Clients.tsx [hunk 4]
```diff
@@ -41,6 +44,7 @@ export default function Clients() {
   };
 
   const handleOpenTankModal = () => {
+    console.log(selectedClient?.firstName, selectedClient?.id);
```
Comment (thomas-harmeyer) [inline frontend/src/pages/dashboardtabs/Clients.tsx]: log

**Review by CarlosMartinezAGV — APPROVED** [review]
Approved for when Thomas' request of removing console.logs is complete

**Pull request merged**